front/src/modules/trainschedule/components/ManageTrainSchedule/SubmitConfAddTrainScheduleV2.tsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+
+import { Plus } from '@osrd-project/ui-icons';
+import { useTranslation } from 'react-i18next';
+import { useSelector } from 'react-redux';
+
+import { osrdEditoastApi } from 'common/api/osrdEditoastApi';
+import type { Infra, TrainScheduleBase } from 'common/api/osrdEditoastApi';
+import { useOsrdConfSelectors } from 'common/osrdContext';
+import { useStoreDataForRollingStockSelector } from 'modules/rollingStock/components/RollingStockSelector/useStoreDataForRollingStockSelector';
+import trainNameWithNum from 'modules/trainschedule/components/ManageTrainSchedule/helpers/trainNameHelper';
+import { setFailure, setSuccess } from 'reducers/main';
+import { useAppDispatch } from 'store';
+import { isoDateToMs, msToIsoDate } from 'utils/date';
+import { castErrorToFailure } from 'utils/error';
+import { sec2time } from 'utils/timeManipulation';
+
+import formatConfV2 from './helpers/formatConfV2';
+
+type SubmitConfAddTrainScheduleProps = {
+  infraState?: Infra['state'];
+  //   refetchTimetable: () => void;
+  //   refetchConflicts: () => void;
+  setIsWorking: (isWorking: boolean) => void;
+  //   setTrainResultsToFetch: (trainScheduleIds?: number[]) => void;
+};
+
+const SubmitConfAddTrainScheduleV2 = ({
+  infraState,
+  //   refetchTimetable,
+  //   refetchConflicts,
+  setIsWorking,
+  //   setTrainResultsToFetch,
+}: SubmitConfAddTrainScheduleProps) => {
+  const [postTrainSchedule] =
+    osrdEditoastApi.endpoints.postV2TimetableByIdTrainSchedule.useMutation();
+  const dispatch = useAppDispatch();
+  const { t } = useTranslation(['operationalStudies/manageTrainSchedule']);
+
+  const { getConf } = useOsrdConfSelectors();
+  const simulationConf = useSelector(getConf);
+  const {
+    pathSteps,
+    timetableID,
+    startTime,
+    trainCount,
+    trainDelta,
+    trainStep,
+    name: confName,
+  } = simulationConf;
```

Comment:
Extract all this data from validConf once you have it
```suggestion
  const {
    pathSteps,
    timetableID,
    startTime,
    trainCount,
    trainDelta,
    trainStep,
    name: confName,
  } = validConf;
```

Reply:
Done